src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/table.tsx

```diff
@@ -0,0 +1,272 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ClipboardTooltip from 'app/components/clipboardTooltip';
+import ExternalLink from 'app/components/links/externalLink';
+import PanelTable from 'app/components/panels/panelTable';
+import QuestionTooltip from 'app/components/questionTooltip';
+import TextOverflow from 'app/components/textOverflow';
+import Tooltip from 'app/components/tooltip';
+import {t, tct} from 'app/locale';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import {BuiltinSymbolSource} from 'app/types/debugFiles';
+import {
+  CandidateDownload,
+  CandidateDownloadStatus,
+  CandidateFeatures,
+  Image,
+} from 'app/types/debugImage';
+
+import NotAvailable from '../notAvailable';
+import Processing, {ProcessingType} from '../processing';
+
+import Feature from './candidate/feature';
+import ProcessingIcon from './candidate/processingIcon';
+import Status from './candidate/status';
+import Actions from './actions';
+import {INTERNAL_SOURCE, onCopy} from './utils';
+
+type Props = {
+  candidates: Image['candidates'];
+  organization: Organization;
+  projectId: Project['id'];
+  baseUrl: string;
+  builtinSymbolSources: Array<BuiltinSymbolSource> | null;
+  onDelete: (debugId: string) => void;
+  isLoading: boolean;
+};
+
+function Table({
+  candidates,
+  organization,
+  projectId,
+  baseUrl,
+  builtinSymbolSources,
+  onDelete,
+  isLoading,
+}: Props) {
+  function renderProcessing(download: CandidateDownload) {
+    const processingItems: React.ComponentProps<typeof Processing>['items'] = [];
+
+    if (download.status !== CandidateDownloadStatus.OK) {
+      return <NotAvailable />;
+    }
+
+    if (download.unwind) {
+      processingItems.push({
+        type: ProcessingType.SYMBOLICATION,
+        icon: <ProcessingIcon processingInfo={download.unwind} />,
+      });
+    }
+
+    if (download.debug) {
+      processingItems.push({
+        type: ProcessingType.STACK_UNWINDING,
+        icon: <ProcessingIcon processingInfo={download.debug} />,
+      });
+    }
+
+    return <Processing items={processingItems} />;
+  }
+
+  function renderFeatures(download: Image['candidates'][0]['download']) {
+    if (download.status !== CandidateDownloadStatus.OK) {
+      return <NotAvailable />;
+    }
+
+    const features = Object.entries(download.features).filter(([_key, value]) => value);
+
+    if (!features.length) {
+      return <NotAvailable />;
+    }
+
+    return (
+      <Features>
+        {Object.entries(download.features)
+          .filter(([_key, value]) => value)
```

Comment:
Isn't this the same as the local var `features`?

Reply:
I don't think so